.github/workflows/snapshot.yml

```
@@ -90,6 +93,7 @@ jobs:
           export QUARKUS_CONTAINER_IMAGE_ADDITIONAL_TAGS=snapshot-${{github.ref_name}}
           export QUARKUS_KUBERNETES_VERSION="${{ env.NEXT_VERSION }}"
           export GIT_REVISION=$(git rev-parse --short HEAD)
+          export QUARKUS_CONTAINER_IMAGE_LABELS="${{ env.QUAY_EXPIRES_AFTER_LABEL }}"
           # Build and push the snapshot images
           mvn -B -P container-image verify --no-transfer-progress -DskipTests \
             -Dquarkus.kubernetes.namespace='$${NAMESPACE}' \
```

I'm not sure the `QUARKUS_CONTAINER_IMAGE_LABELS` env is support for this Quarkus version, and if it is, there are other labels in the app configuration that it could cause to be ignored. Using the system property with `-D` should work, but requires another env at the job level.

```suggestion
          # Build and push the snapshot images
          mvn -B -P container-image verify --no-transfer-progress -DskipTests \
            -Dquarkus.kubernetes.namespace='$${NAMESPACE}' \
            -Dquarkus.container-image.labels."quay.expires-after"=${{ env.IMAGE_EXPIRATION }} \
```